File: LeetCode/Easy/0706-design-hashmap/0706-design-hashmap.java
class MyHashMap {

    int[] values = new int[1_000_001];
    
    public MyHashMap() {
        Arrays.fill(values, -1);
    }
    
    public void put(int key, int value) {
        values[key] = value;
    }
    
    public int get(int key) {
        return values[key];
    }
    
    public void remove(int key) {
        values[key] = -1;
    }
    
}

File: LeetCode/Easy/0706-design-hashmap/0706-design-hashmap.kt
const val MAX_KEY = 1_000_000

class MyHashMap() {

    val values = IntArray(MAX_KEY + 1) { -1 }

    fun put(key: Int, value: Int) {
        values[key] = value
    }

    fun get(key: Int): Int {
        return values[key]
    }

    fun remove(key: Int) {
        values[key] = -1
    }

}